starter-code/javascript/chronometer.js

```diff
@@ -1,40 +1,87 @@
-// Constructor
-// function Chronometer() {
+function Chronometer() {
 
-// }
+    this.currentTime = 0;
+    this.intervalId = 0;
 
-// Chronometer.prototype.startClick = function () {
+}
 
-// };
+Chronometer.prototype.startClick = function () {
 
-// Chronometer.prototype.setMinutes = function () {
-  
-// };
+    var self = this;
 
-// Chronometer.prototype.setSeconds = function () {
-  
-// };
+    this.intervalId = setInterval(function () {
 
-// Chronometer.prototype.twoDigitsNumber = function () {
-  
-// };
+        self.currentTime++;
 
-// Chronometer.prototype.setTime = function () {
+        self.setTime();
 
-// };
+    }, 1000);
 
-// Chronometer.prototype.setMilliseconds = function () {
+};
 
-// };
 
-// Chronometer.prototype.stopClick = function () {
-  
-// };
+Chronometer.prototype.setMinutes = function () {
 
-// Chronometer.prototype.resetClick = function () {
+    this.currentMinutes = 0;
 
-// };
+    return this.currentMinutes = Math.floor(this.currentTime / 60);
```

Comment:
This method will return always a 0 since in the line 25 you have `this.currentMinutes = 0;`